src/common/errors_js.ml

```diff
@@ -106,6 +201,21 @@ let file_of_error err =
   let loc = loc_of_error err in
   match loc.Loc.source with Some filename -> filename | None -> ""
 
+(* TODO: Is this actually needed? Does it make sense with the new format? *)
```

Comment:
We have this so that we can easily diff errors on a large codebase when making changes to Flow's behavior.